asm/Kptfile

```diff
@@ -76,7 +76,7 @@ openAPI:
       x-k8s-cli:
         setter:
           name: anthos.servicemesh.tag
-          value: 1.10.2-asm.3
+          value: 1.10.3-asm.0
```

Comment:
does rev starts with 0 or 1?

Reply:
In proper SemVer I don't know, but I know this is what we named this release :)